### ISSUE TITLE
Ensure keyboard focus is indicated visually

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#55]
Related to #53 

## Description
A well-defined (highly visible) visual indication of keyboard focus needs to be provided for each active element. This position on a screen where an action or keyboard input will take place is referred to as the "focus". Providing a visual indication of the focus allows someone who is viewing the screen to determine what action to take based on what element has focus. This is particularly necessary for keyboard-only users who do not use the mouse and cannot simply click to place focus where they think it should be. The user must rely on the visual indication of focus to determine where an action will occur or determine what keystrokes to perform to move focus to the desired field. If this focus is not indicated the user will be unclear as to where the current execution focus is located and may enter incorrect information, accidentally submit a form without knowledge of what occurred, or spend extra time trying to determine where the keyboard input focus is.

### Button Links
 - [x] Add visual focus indicator with a minimum contrast ratio of 3:1 against stock background colors.  Should use one of the following properties to indicate focus: change of background, lightness, border, outline, underline, and other visual methods.

<img width="742" alt="Screen Shot 2022-02-04 at 3 07 42 PM" src="https://user-images.githubusercontent.com/8166060/152614969-470f56df-a85c-49c5-9344-ae3971affedc.png">

**New Designs**
[This is a dropbox link to updated design for hover & focus states](https://www.dropbox.com/s/5puxqa5nue2lcdl/20220310%20Accessibility%20updates%20to%20buttons.pdf?dl=0).  Note that both the red and green versions of these buttons also have new darker colors to improve color contrast between white button text and button background.  This color change work is captured in #72, but is noted here for convenience.  New red is `#cf4040` and should be used for the value of the `red` scss variable.  New green indicated in the design is `#12726c`, which is extremely similar `turquoise85`, so please just make `turquoise85: #12726c` and use that variable here.

### SlideBlock Controls
 - [x] Add visual focus indicator to desktop controls with a minimum contrast ratio of 3:1 against stock background colors.  Should use one of the following properties to indicate focus: change of background, lightness, border, outline, underline, and other visual methods.
 - [x] Add visual focus indicator to mobile controls with a minimum contrast ratio of 3:1 against stock background colors.  Should use one of the following properties to indicate focus: change of background, lightness, border, outline, underline, and other visual methods.
<img width="475" alt="Screen Shot 2022-02-04 at 3 39 56 PM" src="https://user-images.githubusercontent.com/8166060/152617409-164bc7f4-85c6-4f4b-a390-48ef330f0d23.png">
<img width="1229" alt="Screen Shot 2022-02-04 at 3 39 27 PM" src="https://user-images.githubusercontent.com/8166060/152617412-1393267a-2652-4cb7-b676-1863b48c2821.png">

**New Designs**
[This is a dropbox link to updated design for mobile controls hover & focus states](https://www.dropbox.com/s/kblm682cwojewk1/20220223%20Mobile%20slide%20block.pdf?dl=0) (use the same style for both states).

[This is a dropbox link to updated design for desktop controls hover & focus states](https://www.dropbox.com/s/owakayf7dh012jm/20220223%20Focus%20is%20indicated%20visually.pdf?dl=0) (use the same style for both states).

**Comment by dananjohnson** [comment]
@blake any updates on design for this issue?

**Comment by blnkt** [comment]
@dananjohnson Jose has tickets to update design for some accessibility stuff, including the slider stuff, this sprint.  So we should see those new designs by Wednesday 3/2.

**Comment by blnkt** [comment]
For: **SlideBlock Controls**

[This is a dropbox link to updated design for mobile controls hover & focus states](https://www.dropbox.com/s/kblm682cwojewk1/20220223%20Mobile%20slide%20block.pdf?dl=0) (use the same style for both states).

[This is a dropbox link to updated design for desktop controls hover & focus states](https://www.dropbox.com/s/owakayf7dh012jm/20220223%20Focus%20is%20indicated%20visually.pdf?dl=0) (use the same style for both states).

**Comment by dananjohnson** [comment]
@blnkt thanks for these updated designs! i've address the carousel styles. any updates available yet for the button links?

**Comment by blnkt** [comment]
@dananjohnson nope, not yet.  I would say Jose will probably deliver by 3/4.

**Comment by dananjohnson** [comment]
No problem. Thanks for the update!

**Comment by blnkt** [comment]
New button link hover/focus design added: [This is a dropbox link to updated design for hover & focus states](https://www.dropbox.com/s/5puxqa5nue2lcdl/20220310%20Accessibility%20updates%20to%20buttons.pdf?dl=0). Note that both the red and green versions of these buttons also have new darker colors to improve color contrast between white button text and button background. This color change work is captured in https://github.com/lsst-epo/rubin-obs-client/issues/72, but is noted here for convenience. New red is #cf4040 and should be used for the value of the red scss variable. New green indicated in the design is #12726c, which is extremely similar turquoise85, so please just make turquoise85: #12726c and use that variable here.

**Comment by dananjohnson** [comment]
Resolved by https://github.com/lsst-epo/rubin-obs-client/commit/18b7f2a1af9b68ebe5989c0b3d362338cc057ff3
